packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -364,7 +364,7 @@ async function ssrTransformScript(
           stmt.type !== 'BlockStatement' &&
           stmt.type !== 'ImportDeclaration'
         ) {
-          s.appendRight(stmt.end, ';')
+          s.appendLeft(stmt.end, ';')
```

Comment:
The issue was that ``prependRight(..., `(0,`)`` comes before `appendRight(stmt.end, ';')`, which ends up with a broken code `(0,;`. 

https://github.com/vitejs/vite/blob/e1b520c4fbb3d65a06b04ce8fb3acfa71f253ee9/packages/vite/src/node/ssr/ssrTransform.ts#L402

Reply:
Ahh! Didn't know that was the order there. Great solution 👍🏻